migrations/20221205155420_user_features.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS user_features(
```

Comment:
shouldn't also create the compound key here? where `user` + `feature_name` is unique?